modules/tensor_mechanics/src/materials/ADLAROMANCEStressUpdateBase.C

```diff
@@ -169,350 +237,334 @@ ADLAROMANCEStressUpdateBase::initQpStatefulProperties()
-    Moose::out << "  new immobile dislocations: "
-               << _immobile_old + MetaPhysicL::raw_value(_immobile_dislocation_increment) << "\n"
-               << std::endl;
+    Moose::err << "  old effective strain: " << _effective_strain_old << "\n";
+    Moose::err << "  non-stress extrapolation: "
+               << MetaPhysicL::raw_value(_non_stress_extrapolation) << "\n";
+    Moose::err << "  extrapolation: " << MetaPhysicL::raw_value(_extrapolation[_qp]) << "\n";
+    Moose::err << "  effective strain increment: "
+               << MetaPhysicL::raw_value(rom_effective_strain_inc) << std::endl;
   }
 
-  _creep_rate[_qp] = rom_effective_strain / _dt;
-  _derivative = derivative_rom_effective_strain * -_three_shear_modulus * 1.0e-6 - 1.0;
-
-  return rom_effective_strain - scalar;
-}
+  // Print out relevant verbose information, and quit creep stress calculation
+  if (!_extrapolation[_qp])
+  {
+    if (scalar)
+      mooseError("In ", _name, ": Internal error: Scalar (", scalar, ") isn't zero!");
+    if (_verbose)
+      Moose::err << "    ROM evalulation skipped due to out-of-range input!" << std::endl;
+
+    _creep_rate[_qp] = 0.0;
+    _derivative = 1.0;
+    return 0.0;
+  }
 
-void
-ADLAROMANCEStressUpdateBase::computeStressFinalize(const ADRankTwoTensor & plastic_strain_increment)
-{
-  _mobile_dislocations[_qp] = _mobile_old + _mobile_dislocation_increment;
-  _immobile_dislocations[_qp] = _immobile_old + _immobile_dislocation_increment;
+  _creep_rate[_qp] = rom_effective_strain_inc / _dt;
+  _derivative = drom_effective_strain_inc_dstress * dtrial_stress_dscalar - 1.0;
 
-  if (_verbose)
+  if (!_apply_strain)
   {
-    Moose::out << "Finalized verbose information from " << _name << "\n";
-    Moose::out << "  increment effective creep strain: "
-               << std::sqrt(2.0 / 3.0 *
-                            MetaPhysicL::raw_value(plastic_strain_increment.doubleContraction(
-                                plastic_strain_increment)))
-               << "\n";
-    Moose::out << "  effective_creep_strain: "
-               << std::sqrt(2.0 / 3.0 *
-                            MetaPhysicL::raw_value(
-                                _creep_strain[_qp].doubleContraction(_creep_strain[_qp])))
-               << "\n";
-    Moose::out << "  new mobile dislocations: " << MetaPhysicL::raw_value(_mobile_dislocations[_qp])
-               << "\n";
-    Moose::out << "  new immobile dislocations: "
-               << MetaPhysicL::raw_value(_immobile_dislocations[_qp]) << "\n"
-               << std::endl;
+    if (_verbose)
+      Moose::err << "    Strain not applied due to apply_strain input parameter!" << std::endl;
+    return 0.0;
   }
-
-  ADRadialReturnCreepStressUpdateBase::computeStressFinalize(plastic_strain_increment);
+  return rom_effective_strain_inc - scalar;
 }
 
-Real
-ADLAROMANCEStressUpdateBase::computeTimeStepLimit()
```

Comment:
It's always weird that github does this...I just moved computeTimeStepLimit towards the bottom...